dashboard/config/locales/failure_message_overrides.el-GR.yml

```diff
@@ -1,85 +1,2 @@
-      Kiki_StarWars_Test_5_Example_failure_message_override: Game over!
-      MC HOC 2016 Level 2-2_failure_message_override: Use `move forward` and `turn`
-        commands to get your chicken moving.
-      MC HOC 2016 Level 3_failure_message_override: Use a `forever do` block and `move
-        forward` and `turn` commands to keep your chicken wandering.
-      MC HOC 2016 Level 4_failure_message_override: If the chickens are all moving
-        in the same direction, use `turn random` to make the chickens turn randomly
-        left or right.
-      MC HOC 2016 Level 5_failure_message_override: Try using 'Forever Do' with 'Move
-        a Step Toward the Player'.
-      MC HOC 2016 Level 6_failure_message_override: Use the `spawn zombie` command
-        under `when night` to spawn at least three zombies. Then, add `move a step
-        toward player` to their `when spawned` event with a loop to help them chase
-        you. If they’re too fast, add a `wait` command to slow them down.
-      MC HOC 2016 Level 8_failure_message_override: You need to use the exploding
-        Creeper to blow up the wall so you can reach your sheep.
-      MC HOC 2016 New Drop_failure_message_override: Use the arrow keys to walk to
-        the sheep and press space bar or click the game board to use the sheep. And,
-        don’t forget to add the 'Drop' block to your code.
-      MC HOC 2016 New Spawn_failure_message_override: Place `spawn sheep` under `when
-        run`.
-      MC HOC 2016 New Walk_failure_message_override: To open the door, press the space
-        bar or touch the game board while facing it. Walk inside to continue.
-      MC HOC 2016 SCORE LEVEL_failure_message_override: If you're having trouble tagging
-        5 Golems, try changing the score to 2 per tag.
-      MC HOC 2016 Village Test_failure_message_override: You need to make the Iron
-        Golem chase and attack the zombies to win.
-      Valentine_playlab_01_failure_message_override: Almost!  Try again.
-      WhileAdventure_failure_message_override: Congratulations!  You gathered X coins!
-      courseE_playLab_scaffold3_failure_message_override: You must add two or more
-        new actors to your game and program them to move.
-      courseE_playLab_scaffold3_2018_failure_message_override: You must add two or
-        more new actors to your game and program them to move.
-      grade4_scaffolded_project3_failure_message_override: You must add two or more
-        new actors that move to your game.
-      kikiTesta_failure_message_override: Congratulations!  You gathered X coins!
-      sample1_failure_message_override: Congratulations!  You gathered X coins!
-      sample4_failure_message_override: Congratulations!  You gathered X coins!
-      sample6_failure_message_override: Not quite.  Keep trying!  [Move to Play Zone]
-      sample8_failure_message_override: Congratulations!  You gathered X coins!
+el: 
```

Comment:
This file went blank. Guessing this is fine since we didn't lose any translations and we fallback to the source `en` file?

Reply:
Yep! Ram did a bunch of work last week to verify that fallbacks should always work